transport/wsr/src/test/java/org/kaazing/gateway/transport/wsr/WsrAcceptorTest.java

```diff
@@ -29,12 +29,18 @@
 import java.util.Collections;
```

Comment:
Please just remove this file altogether since we are no longer supporting wsr. You can close the bug (issue #287) too.